cmd/utils.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// ProjectType ...
+type ProjectType string
+
+const (
+	// iOS
+	xcodeWorkspace ProjectType = "*.xcworkspace"
+	xcodeProject   ProjectType = "*.xcodeproj"
+)
+
+// Scans the root dir for the provided project files
+// If none of them in the root dir, then it will return an error
+func scanForProjectFiles(projectFiles []ProjectType) (string, error) {
```

Comment:
here we expect `projectFiles` however it will be N project file extension, the naming is very mixed